CMakeLists.txt

```diff
@@ -15,54 +15,91 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "GNU" OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
-find_package(GDAL 3 REQUIRED)
 find_package(ament_cmake REQUIRED)
+find_package(grid_map_msgs REQUIRED)
 find_package(grid_map_ros REQUIRED)
 find_package(grid_map_core REQUIRED)
+find_package(rclcpp REQUIRED)
+
 find_package(Eigen3 REQUIRED)
+find_package(GDAL 3 REQUIRED)
 
-add_library(${PROJECT_NAME})
-add_subdirectory(src)
-
-target_include_directories(${PROJECT_NAME}
-    PUBLIC
-    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
-    "$<INSTALL_INTERFACE:include/${PROJECT_NAME}>"
-)
+find_package(yaml_cpp_vendor REQUIRED)
+link_directories(${yaml_cpp_vendor_LIBRARY_DIRS})
```

Comment:
Where did you find this variable?
https://github.com/search?q=repo%3Aros2%2Fyaml_cpp_vendor%20yaml_cpp_vendor_LIBRARY_DIRS&type=code

Reply:
Details are in:
https://github.com/ament/ament_cmake/blob/rolling/ament_cmake_target_dependencies/cmake/ament_target_dependencies.cmake